File: strings/front-3--slice-check-repeat-concatenate.js
function frontThree(str) {
	return str.substr(0, 3).repeat(3)
}

frontThree('Python') // 'PytPytPyt', 'Simple string#1'
frontThree('Chocolate') // 'ChoChoCho', 'Simple strin#2'
frontThree('duh') // 'duhduhduh', '3 characters string'
frontThree('Sportsmanship') // 'SpoSpoSpo', 'Generic string'
frontThree('ab') // 'ababab', '2 characters string'
frontThree('a') // 'aaa', '1 characters string'
frontThree('') // '', 'Empty string'

File: strings/front-3--slice-check-repeat-concatenate.py
def front3(txt):
	return txt[0:3] * 3

front3('Python') # 'PytPytPyt', 'Simple string#1'
front3('Chocolate') # 'ChoChoCho', 'Simple strin#2'
front3('duh') # 'duhduhduh', '3 characters string'
front3('Sportsmanship') # 'SpoSpoSpo', 'Generic string'
front3('ab') # 'ababab', '2 characters string'
front3('a') # 'aaa', '1 characters string'
front3('') # '', 'Empty string'